src/Ng.TraceListeners/AzureTableStorageStatusTraceListener.cs

```diff
@@ -0,0 +1,134 @@
+        }
+
+        protected override void TraceSimpleEvent(DateTime eventTime, Int32 threadId, TraceEventType eventType, String message)
+        {
+            // Called by Trace.WriteLne(). Ignore.
+        }
+
+        protected override void TraceSimpleEvent(DateTime eventTime, Int32 threadId, TraceEventType eventType, String message, Int32 eventId, TraceEventCache eventCache, String source)
+        {
+            if (_table != null)
+            {
+                if (message.StartsWith("#Status ") || message.StartsWith("#Status:"))
+                {
+                    Status status = new Status(eventTime.ToUniversalTime());
+                    status.Level = eventType.ToString();
+                    status.ThreadId = threadId;
+
+                    string trimmedMessage = message.Substring(8);
+
+                    string[] parts = trimmedMessage.Split(new[] { '#' }, StringSplitOptions.RemoveEmptyEntries);
+
+                    foreach (string part in parts)
+                    {
+                        int propertyNameIndex = part.IndexOf('=');
+
+                        if (propertyNameIndex > 0)
+                        {
+                            string propertyName = part.Substring(0, propertyNameIndex);
+                            string propertyValue = part.Substring(propertyNameIndex + 1);
+
+                            switch (propertyName.ToLowerInvariant())
+                            {
+                                case "activity":
+                                    status.Activity = propertyValue;
+                                    break;
+                                case "state":
+                                    status.State = propertyValue;
+                                    break;
+                                case "result":
+                                    status.Result = propertyValue;
+                                    break;
+                                case "details":
+                                    if (!string.IsNullOrWhiteSpace(status.Details))
+                                    {
+                                        status.Details += "; ";
+                                    }
+
+                                    status.Details += propertyValue;
+                                    break;
+                                default:
+                                    throw new InvalidOperationException($"Unknown trace property name: {propertyName}");
+                            }
+                        }
+                        else
+                        {
+                            if (!string.IsNullOrWhiteSpace(status.Details))
+                            {
+                                status.Details += "; ";
+                            }
+
+                            status.Details += part;
+                        }
+                    }
+
+                    TableOperation operation = TableOperation.Insert(status);
```

Comment:
Yes, we currently log the following values to the table storage.
These ones that are set automatically. i.e. for the most part, you always get them.
- Machine
- EventTime
- Application
- ProcessId
- ThreadId
- Message Level (Information, Warning, Error)

These are the ones that are set by the trace message. i.e. they're freeform strings set in the trace message. BTW, I'm not sure these are the correct fields to include. But since they're freeform strings, they're ok for now. 

- Activity
- State
- Result
- Details

Eventually, it would be nice if we had a concept of a RunId. i.e. a value that all the tools use for a particular run. That way we could view the logs for one run on one page.